zhaquirks/__init__.py

```diff
@@ -66,18 +66,37 @@ class LocalDataCluster(CustomCluster):
 
     async def bind(self):
         """Prevent bind."""
+        self.log(
+            logging.DEBUG,
```

Comment:
Also, do we want to use the zigpy logger here or create a quirks logger? When we use the zigpy one, I think we always have a zigpy prefix in logs, not a zhaquirks one, so it's sometimes a bit hard to differentiate what's coming from where.
It's not a huge issue here, as we can clearly identify it as coming from quirks, but it's an issue with other quirks as well (that use the zigpy logger with generic messages, so it's unclear where they're coming from).
We may want to standardize our approach to this at some point.

Reply:
IMO the log is from a cluster impl so it should use the cluster logger but I am open to discussing some sort of standardization